packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -11,33 +11,43 @@ import { IconButton, Toolbar } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { addQueryArgs } from '@wordpress/url';
 
-function FullscreenModeClose( { isActive, postType } ) {
-	if ( ! isActive || ! postType ) {
+function FullscreenModeClose( { isActive, postType, isAlwaysShown, isLabeled } ) {
+	if ( ! ( isActive || isAlwaysShown ) || ! postType ) {
 		return null;
 	}
 
+	const label = get(
+		postType,
+		[ 'labels', 'view_items' ],
```

Comment:
In the future, it would be cool to add another way to extend what label/href is used!

Reply:
Definitely!  I will have to do some more digging to see what the best option for doing this might be.  Are you thinking carving out a spot in the store where we could set overrides for the label / href?

Reply:
I'm not sure what the best solution would be 🤔